docs/commands.md

```diff
@@ -56,6 +56,17 @@ Allows overriding whether bundle is minified. This defaults to false if dev is t
 
 File name where to store the resulting bundle, ex. `/tmp/groups.bundle`.
 
+If you are planning on building an APK, that will run without the packager, by invoking `./gradlew assembleDebug` you have to make sure to put the bundle into the right directory and give it the right name, so that gradle can find it.
+
+For react-native versions 0.57 and above the bundle output path should be:
+`android/app/build/generated/assets/react/debug/index.android.js`
+
+To find out the correct path for previous react-native versions, take a look at the `react.gradle` file here: https://github.com/facebook/react-native/blob/0.57-stable/react.gradle or inside your `node_modules/react-native` directory.
+
+The expected path for the js bundle can be found on the line that starts with `jsBundleDir = `.
+
+Alternatively you can simply set `bundleInDebug: true` in your app/build.gradle file, inside the `project.ext.react` map.
```

Comment:
I feel like we should highlight this fact and hide the rest behind a `<detail>` element